privaterelay/utils.py

```diff
@@ -0,0 +1,37 @@
+from django.conf import settings
+
+def get_premium_countries_info_from_request(request):
+    country_code = _get_cc_from_request(request)
+    premium_countries = settings.PREMIUM_PLAN_COUNTRY_LANG_MAPPING.keys()
+    premium_available_in_country = country_code in premium_countries
+    return({
+        'country_code': country_code,
+        'premium_countries': premium_countries,
+        'premium_available_in_country': premium_available_in_country
+    })
+
+
+def _get_cc_from_request(request):
+    if 'X-Client-Region' in request.headers:
+        return request.headers['X-Client-Region'].lower()
+    if 'Accept-Language' in request.headers:
+        return get_premium_country_lang(request.headers['Accept-Language'])[0]
+    if settings.DEBUG:
+        return 'us'
+    return 'us'
+
+
+def get_premium_country_lang(accept_lang, cc=None):
+    lang = accept_lang.split(',')[0]
+    lang_parts = lang.split("-") if lang and "-" in lang else [lang]
+    lang = lang_parts[0].lower()
+    if cc is None:
+        cc = lang_parts[1] if len(lang_parts) == 2 else lang_parts[0]
+        cc = cc.lower()
+
+    if cc in settings.PREMIUM_PLAN_COUNTRY_LANG_MAPPING.keys():
+        languages = settings.PREMIUM_PLAN_COUNTRY_LANG_MAPPING[cc]
+        if lang in languages.keys():
+            return cc, lang
+        return cc, list(languages.keys())[0]
+    return cc, 'en'
```

Comment:
@groovecoder Hmm, looks like we have other code that assumes that this function only returns country codes that are present in `PREMIUM_PLAN_COUNTRY_MAPPING`, e.g. here: https://github.com/mozilla/fx-private-relay/blob/68c209d2c447c5fbe69a2c4216c8685bccc51b8d/privaterelay/templatetags/relay_tags.py#L57

If I run this and set my browser's Accept-Language to `ro-ro`, I get a KeyError because `ro` is not present there :/

Reply:
argh!!! okay let me add a test case and fix.

Reply:
Okay, I updated the `relay_tags` to return sensible values when the country code isn't in the `MAPPING`. (I think I checked that all places that need the ID will work with a `''` value, and everywhere that needs a price can at least *render* a USD price instead.

Reply:
Testing it locally, I'm no longer encountering any problems! :tada: